src/main/java/org/folio/gobi/Mapper.java

```diff
@@ -720,18 +720,18 @@ private void mapLicense(List<CompletableFuture<?>> futures, License license, Doc
 
   private void mapFundDistribution(List<CompletableFuture<?>> futures, FundDistribution fundDistribution, Document doc,
                                    PostGobiOrdersHelper postGobiOrdersHelper) {
+    Optional.ofNullable(mappings.get(Mapping.Field.FUND_CODE))
```

Comment:
It makes sense to map fund code only when fundId can be mapped. So this piece of code should be moved back

Reply:
can we make **" map fund id only when fundCode can be mapped"**?? as fundCode is also a mendatory field. Kindly suggest.